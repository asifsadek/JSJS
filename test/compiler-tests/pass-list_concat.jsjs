val nums = [1, 2, 3, 4, 5];

print(List.concat(nums, [7]));

val strings = ["one", "two", "three", "four", "five"];

print(List.concat(strings, ["seven"]));

val bools = [true, false, false, true, false];

print(List.concat(bools, [false]));
