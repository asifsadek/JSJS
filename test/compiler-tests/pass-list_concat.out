List [ 1, 2, 3, 4, 5, 7 ]
List [ "one", "two", "three", "four", "five", "seven" ]
List [ true, false, false, true, false, false ]
